metadata.json

```diff
@@ -8,10 +8,72 @@
   "project_page": "https://github.com/puppetlabs/puppetlabs-rook",
   "issues_url": "https://github.com/puppetlabs/puppetlabs-rook/issues",
   "dependencies": [
-    {"name":"puppetlabs-stdlib","version_requirement":">= 4.24.0"},
-    {"name":"puppetlabs/translate","version_requirement":">= 0.0.1 < 1.0.0"},
-    {"name":"stahnma/epel","version_requirement":">= 1.2.2 < 1.3.0"}
+    {
+      "name": "puppetlabs-stdlib",
+      "version_requirement": ">= 4.24.0 < 6.0.0"
+    },
+    {
+      "name": "puppetlabs/translate",
+      "version_requirement": ">= 0.0.1 < 2.0.0"
+    },
+    {
+      "name": "stahnma/epel",
+      "version_requirement": ">= 1.2.2 < 1.3.0"
+    }
   ],
-  "data_provider": null
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "OracleLinux",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Scientific",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+        "8"
+      ]
+    },
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "16.04"
+      ]
+    },
+    {
+      "operatingsystem": "windows",
+      "operatingsystemrelease": [
+        "2008 R2",
+        "2012 R2",
+        "10"
+      ]
+    }
+  ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 4.7.0 < 6.0.0"
+    }
+  ],
+  "pdk-version": "1.7.0",
+  "template-url": "file:///opt/puppetlabs/pdk/share/cache/pdk-templates.git",
```

Comment:
Can you try running the convert again, but using the `--template-url=https://github.com/puppetlabs/pdk-templates` flag? We recommend that internal modules use the github repo instead of the packaged templates so that you can inherent changes to the templates faster than potential PDK release cadence.

Reply:
@bmjen sure will try now

Reply:
@bmjen it works.The only difference in metadata.json file which is 77c77
<   "template-url": "file:///opt/puppetlabs/pdk/share/cache/pdk-templates.git",
---
>   "template-url": "https://github.com/puppetlabs/pdk-templates",

as expected.

Reply:
The only difference in metadata.json file which is 77c77
   "template-url": "file:///opt/puppetlabs/pdk/share/cache/pdk-templates.git",
- - -
   "template-url": "https://github.com/puppetlabs/pdk-templates",

Reply:
@bmjen committed the changes with pdk convert using template